package.json

```diff
@@ -78,5 +88,33 @@
     "vinyl-source-stream": "^1.1.0",
     "watchify": "^3.8.0",
     "web-ext": "^1.8.1"
+  },
+  "jest": {
+    "globals": {
+      "browser": {}
+    },
+    "testMatch": [
+      "<rootDir>/src/**/*.test.(js|jsx)"
+    ],
+    "rootDir": ".",
+    "transform": {
+      "^.+\\.(js|jsx)$": "<rootDir>/node_modules/babel-jest"
+    },
+    "modulePaths": [
+      "<rootDir>"
+    ],
+    "moduleNameMapper": {
+      "\\.(css|less)$": "identity-obj-proxy"
+    },
+    "moduleFileExtensions": [
+      "js",
+      "jsx"
+    ],
+    "unmockedModulePathPatterns": [
+      "<rootDir>/node_modules/react",
+      "<rootDir>/node_modules/react-dom",
+      "<rootDir>/node_modules/react-addons-test-utils",
+      "<rootDir>/node_modules/fbjs"
+    ]
```

Comment:
I'd keep this config in a separate file, say `.jest-config.json`, and pass that with `--config <path/to/json>` ([docs](https://facebook.github.io/jest/docs/configuration.html#selector(exact=--config%20%3Cpath/to/json%3E,type=TextQuoteSelector)))